bin/install_cmdstan

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
```

Comment:
Could this file have `.py` extension? It still can be called without `.py` on linux

Reply:
doesn't seem to work on Mac OS.